gp-inventory/gpi-inventory-shortcode.php

```diff
@@ -1,236 +1,5 @@
 <?php
 /**
  * WARNING! THIS SNIPPET IS DEPRECATED. 🚧
- *
- * Gravity Perks // Inventory // Inventory Shortcode
- * https://gravitywiz.com/documentation/gravity-forms-inventory/
- *
- * Instruction Video: https://www.loom.com/share/9d1897075b7b435389479d4f17fc6807
- *
- * Update "123" to your form ID and "4" to your field ID with inventory.
- *
- * `[gravityforms action="inventory" id="123" field="4"]`
- *
- * If using a field with scopes, you may provide the scope values in a comma-delimited list:
- *
- * `[gravityforms action="inventory" id="123" field="4" scope_values="Scope Value,Another Scope Value"]`
- *
- * This snippet requires GP Inventory 1.0-beta-1.7 or newer.
- *
- * @todo
- * - Add support for excluding field parameter and showing a consolidated list of all inventories.
+ * This snippet's functionality has been moved to the GP Inventory plugin in version 1.0.
```

Comment:
```suggestion
 * This snippet's functionality has been moved to [GP Inventory][1] in version 1.0.
 *
 * [1] https://gravitywiz.com/documentation/gravity-forms-inventory/ 
```

Felt like a link would be nice to redirect them more efficiently. No strong feelings.

Reply:
I dig it! And Code Rabbit agrees 😆 

<img width="852" alt="image" src="https://github.com/user-attachments/assets/40473097-f508-470e-a8b5-a0dc016fe629" />

Reply:
Done ✅